### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by milla7** [MountainGod2/chaturbate_poller#3] [General]
Hi! contact me please, XXXXXXX@gmail.com +XXXXXXXXXX whatsapp

**Comment by MountainGod2** [comment]
Is this something I can help you with here? Would prefer to keep discussions related to this project on Github if possible 😄

**Comment by milla7** [comment]
> Is this something I can help you with here? Would prefer to keep discussions related to this project on Github if possible 😄

I want to develop a functionality, I have an Arduino controlled toy created by me, I want to see if it is possible to connect it with chaturbate

**Comment by MountainGod2** [comment]
Assuming you already have a way to comminicate with the Arduino, the [` tip_handler `](https://github.com/MountainGod2/chaturbate_poller/blob/7b132059c34d92c6d3be97ae656bbbf18e25d7a3/examples/event_handler.py#L17) function in `examples/event_handler.py` might help provide some insight on how you could potentially create actions based off the responses from the API.

Let me know if you have any questions!
